Library/ShareContext.swift

```diff
@@ -62,3 +65,25 @@ public enum ShareContext {
     }
   }
 }
+
+extension ShareContext: Equatable {
+}
+
+public func == (lhs: ShareContext, rhs: ShareContext) -> Bool {
```

Comment:
I think the modern swift way of conforming to equatable is to put the `func ==` inside the extension as a `static func ==`